packages/toolpad-core/package.json

```diff
@@ -18,7 +18,15 @@
   "keywords": [
     "react",
     "mui",
-    "toolpad"
+    "toolpad",
+    "internal tools",
+    "typescript",
+    "crud",
+    "self hosted",
+    "admin panel",
```

Comment:
How do you feel about replacing "admin panel", "admin framework" and "react admin dashboard" with just "admin" and "dashboard"?